kratos/CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ set( KRATOS_CORE_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/modified_shape_functions/triangle_2d_3_ausas_modified_shape_functions.cpp;
     ${CMAKE_CURRENT_SOURCE_DIR}/modified_shape_functions/tetrahedra_3d_4_modified_shape_functions.cpp;
     ${CMAKE_CURRENT_SOURCE_DIR}/modified_shape_functions/tetrahedra_3d_4_ausas_modified_shape_functions.cpp;
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/test_element.cpp;
```

Comment:
Please remove this. This is already added in line 131 and leaving it here will cause duplication in several steps during compilation

Reply:
Do you mean that all the cpp in test folder are detected automatically?

Reply:
Yes